.github/CODEOWNERS

```
@@ -32,7 +32,7 @@
 /cloud-sql/**/*                        @GoogleCloudPlatform/infra-db-dpes @GoogleCloudPlatform/python-samples-reviewers
 /cloud_tasks/**/*                      @GoogleCloudPlatform/infra-db-dpes @GoogleCloudPlatform/python-samples-reviewers
 /codelabs/**/*                         @GoogleCloudPlatform/python-samples-reviewers
-/composer/**/*                         @leahecole @rachael-ds @rafalbiegacz @GoogleCloudPlatform/python-samples-reviewers
+/composer/**/*                         @leahecole @rachael-ds @cloud-dpes-composer @GoogleCloudPlatform/python-samples-reviewers
```

Is it fine to remove you both, now that you're both covered by the team? Or would you still like explicit access
```suggestion
/composer/**/*                         @GoogleCloudPlatform/cloud-dpes-composer @GoogleCloudPlatform/python-samples-reviewers
```

---

I left it as explicit access for now because I didn't update blunderbuss to match - I wanted Rachael and I to be the first line of defense in general but to make sure folks in this team could approve PRs

---

Acknowledged.